src/components/ChatHeader/ChatHeader.jsx

```diff
@@ -1,59 +1,59 @@
-import { useState } from 'react';
-import PropTypes from 'prop-types';
 import { useSelector } from 'react-redux';
-import { getUser } from '@/redux-store/selectors.js';
-import CountryInfo from '../CountryInfo/CountryInfo';
+import { useState } from 'react';
+import { getUser } from '@/redux-store/selectors';
 import { ChatHeaderStyled } from './ChatHeaderStyled';
+import CountryInfo from '../CountryInfo/CountryInfo';
 
 const ChatHeader = ({
-  countryName = 'Country Name',
-  participantsAmount = 0,
-  countryChatId,
-  setSubscriptionCountryRooms,
-  isSubscribed,
+  chatName,
+  participantsAmount,
+  selectedCompanion,
+  isPrivateChat,
   isUserTyping,
   userNameisTyping,
+  chatId,
+  isSubscribed,
+  setSubscriptionRooms,
 }) => {
   const [openModal, setOpenModal] = useState(false);
   const userName = useSelector(getUser)?.userName;
-  const handleOpen = () => setOpenModal(true);
+  const showUserIsTyping = isUserTyping && userNameisTyping !== userName;
+
+  const handleOpen = () => {
+    if (!isPrivateChat) {
+      setOpenModal(true);
+    }
+  };
   const handleClose = () => {
     setOpenModal(false);
   };
 
-  const showUserIsTyping = isUserTyping && userNameisTyping !== userName;
-
   return (
     <>
       <ChatHeaderStyled onClick={handleOpen}>
-        <h5>{countryName}</h5>
+        <h5>{isPrivateChat ? selectedCompanion.userName : chatName}</h5>
+
         <p>
           {showUserIsTyping
             ? `${userNameisTyping} is typing...`
-            : `${participantsAmount} participants`}
+            : !isPrivateChat && `${participantsAmount || '0'} participants`}
         </p>
       </ChatHeaderStyled>
-      <CountryInfo
-        isOpen={openModal}
-        onClose={handleClose}
-        countryName={countryName}
-        participantsAmount={participantsAmount}
-        countryChatId={countryChatId}
-        setSubscriptionCountryRooms={setSubscriptionCountryRooms}
-        isSubscribed={isSubscribed}
-      />
+      {!isPrivateChat && (
+        <CountryInfo
+          isOpen={openModal}
+          onClose={handleClose}
+          countryName={chatName || 'Country Name'}
+          participantsAmount={participantsAmount || 0}
+          chatId={chatId}
+          setSubscriptionRooms={setSubscriptionRooms}
+          isSubscribed={isSubscribed}
+        />
+      )}
     </>
   );
 };
 
-ChatHeader.propTypes = {
```

Comment:
Why the PropTypes was removed?

Reply:
Oops. I think i missed this when i resolve conflicts. I returned